README.md

```diff
@@ -149,7 +149,7 @@ In order to submit a job to a remote cluster, you simply run the following code:
 
 ```python
 # Define the distributed optimization procedure, and its parameters.
-trainer = ADAG(keras_model=mlp, worker_optimizer=optimizer_mlp, loss=loss_mlp, num_workers=20,
+trainer = ADAG(keras_model=mlp, worker_optimizer=optimizer_mlp, loss=loss_mlp, metrics = ["accuracy"], num_workers=20,
```

Comment:
In order to preserve consistency, could you add "accuracy" as the default metric for all optimizers?